test/RelayPricer.test.ts

```diff
@@ -49,8 +54,8 @@ describe('RelayPricer', () => {
     });
 
     describe('getExchangeRate', () => {
-        let fakeSourceApi: StubbedInstance<ExchangeApi>;
-        const conversionError = `Currency conversion for pair ${sourceCurrency}:${targetCurrency} not found in current exchange api`;
+        let fakeSourceApi: StubbedInstance<BaseExchangeApi>;
+        const CONVERSION_ERROR = `Currency conversion for pair ${sourceCurrency}:${targetCurrency} not found in current exchange api`;
```

Comment:
NIT: I know it's a matter of preferences, but I tend to name variables with UPPER_CASE only if they have a constant value and IMHO it doesn't apply in this case.

Reply:
Done.